src/Stripe.net/Services/Account/AccountService.cs

```diff
@@ -58,12 +58,12 @@ public virtual Task<Account> GetAsync(string accountId, RequestOptions requestOp
 
         public virtual Account GetSelf(RequestOptions requestOptions = null)
         {
-            return this.GetRequest<Account>($"{Urls.BaseUrl}/account", null, requestOptions, false);
+            return this.GetRequest<Account>($"{StripeConfiguration.ApiBase}/v1/account", null, requestOptions, false);
         }
 
         public virtual Task<Account> GetSelfAsync(RequestOptions requestOptions = null, CancellationToken cancellationToken = default(CancellationToken))
         {
-            return this.GetRequestAsync<Account>($"{Urls.BaseUrl}/account", null, requestOptions, false, cancellationToken);
+            return this.GetRequestAsync<Account>($"{StripeConfiguration.ApiBase}/v1/account", null, requestOptions, false, cancellationToken);
```

Comment:
ooc can't we do this in `GetRequestAsync` instead?

Reply:
Yup! I plan on doing exactly that in another PR.